src/Microsoft.DotNet.Build.Tasks/GenerateEncodingTable.cs

```diff
@@ -0,0 +1,395 @@
+﻿using Microsoft.Build.Framework;
```

Comment:
by the way, your changes LGTM. just wait for @weshaggard to have a quick look too and we should be good to go.